integration/setup/setup.go

```diff
@@ -61,7 +61,7 @@ var (
 
 	disableFilterPushdownF = flag.Bool("disable-filter-pushdown", false, "disable filter pushdown")
 	enableSortPushdownF    = flag.Bool("enable-sort-pushdown", false, "enable sort pushdown")
-	enableOplogF           = flag.Bool("enable-oplog", false, "enable OpLog")
+	enableOplogF           = flag.Bool("enable-oplog", true, "enable OpLog")
```

Comment:
@AlekSi after I enabled `oplog` in tests, some tests that check collections lists started failing because FerretDB produces a collection called `oplog` in addition to "regular" collection. Do you have any advice regarding that?

Reply:
#3625

Reply:
Let's drop that flag and always enable OpLog

Reply:
Fixed (ready for another round of review!)